docs/events.md

```diff
@@ -0,0 +1,88 @@
+|  LOAD_START| loadstart|Fires when the browser starts looking for the audio/video
+|  PAUSE| pause |Fires when the audio/video has been paused
+|PLAY  | play |Fires when the audio/video has been started or is no longer paused
+|  PLAYING| playing |Fires when the audio/video is playing after having been paused or stopped for buffering
+| PROGRESS | progress|Fires when the browser is downloading the audio/video
+| RATE_CHANGE | ratechange |Fires when the playing speed of the audio/video is changed
+| SEEKED | seeked |Fires when the user is finished moving/skipping to a new position in the audio/video
+|  SEEKING| seeking |Fires when the user starts moving/skipping to a new position in the audio/video
+| STALLED | stalled |Fires when the browser is trying to get media data, but data is not available
+| SUSPEND |  suspend| Fires when the browser is intentionally not getting media data
+| TIME_UPDATE| timeupdate |Fires when the current playback position has changed
+| VOLUME_CHANGE | volumechange |Fires when the volume has been changed
+| WAITING |  waiting| Fires when the video stops because it needs to buffer the next frame
+
+## Custom events list
+|  Event Name| Event String | Description |
+|--|--|--|
+| ENTER_FULLSCREEN | enterfullscreen| Fires when the player enters fullscreen |
+| EXIT_FULLSCREEN | exitfullscreen | Fires when the player exits fullscreen |
+| REQUESTED_ENTER_FULLSCREEN | requestedenterfullscreen | Fires when the player received a request to enter fullscreen |
+| REQUESTED_EXIT_FULLSCREEN |  requestedexitfullscreen|Fires when the player received a request to exit fullscreen  |
+| AUTOPLAY_FAILED | autoplayfailed | Fires when browser fails to autoplay with sound |
+| FALLBACK_TO_MUTED_AUTOPLAY | fallbacktomutedautoplay | Fires when browser fails to autoplay with sound but start muted autoplay instead |
+| CHANGE_SOURCE_STARTED | changesourcestarted | Fires when change source flow started |
+| CHANGE_SOURCE_ENDED |  changesourceended| Fires when change source flow ended |
+| MUTE_CHANGE | mutechange | Fires when the volume has been muted/unmute |
+| VIDEO_TRACK_CHANGED | videotrackchanged | Fires when the active video track has been changed |
+| AUDIO_TRACK_CHANGED | audiotrackchanged | Fires when the active audio track has been changed |
+| TEXT_TRACK_CHANGED | texttrackchanged | Fires when the active text track has been changed |
+| TEXT_CUE_CHANGED | textcuechanged |Fires when the active text track cue has changed  |
+| TRACKS_CHANGED | trackschanged | Fires when the player tracks have been changed |
+| ABR_MODE_CHANGED | abrmodechanged | Fires when the abr mode change from 'auto' to 'manual' or vice versa|
+| ABR_MODE_CHANGED | abrmodechanged | Fires when the abr mode change from 'auto' to 'manual' or vice versa |
+| PLAYER_STATE_CHANGED | playerstatechanged | Fires when the player state has been changed |
+|FIRST_PLAY  |  firstplay| Fires on the first play |
```

Comment:
See if you are ok with this change:
Fires when the asset is first played.

Reply:
yes